content/docs/setup/kubernetes/additional-setup/config-profiles/index.md

```diff
@@ -26,8 +26,8 @@ your specific needs. The following built-in configuration profiles are currently
 
 1. **minimal**: the minimal set of components necessary to use Istio's [traffic management](/docs/tasks/traffic-management/) features.
 
-1. **sds**: similar to the **default** profile, but also enables Istio's [SDS (secret discovery service)](/docs/tasks/security/auth-sds).
-    This profile comes only with authentication enabled.
+1. **sds-auth**: similar to the **default** profile, but also enables Istio's [SDS (secret discovery service)](/docs/tasks/security/auth-sds).
+    This profile comes with additional authentication features enabled by default (see table below).
```

Comment:
The '(see table below)' is probably not ideal. Is this necessary, or should I just omit it and leave the user to naturally find out later when they read further?

Reply:
agreed.  I'd remove the ` (see table below)`